Library/Homebrew/formula.rb

```diff
@@ -2219,7 +2219,17 @@ def method_added(method)
     # Shows when running `brew info`.
```

Comment:
```suggestion
    # Shows when running `brew info`.
    # Multiple licenses means that the software is licensed under multiple licenses.
    # Do not use multiple licenses if e.g. different parts are under different licenses.
```

Reply:
Thanks @MikeMcQuaid sorry you had to make a bunch of changes for trivial stuff like that. My aim was more to see if the direction was right! Will refine future commits!

Reply:
No problem! I said I'd help with the wording, this isn't trivial!